openslides/motions/static/js/motions/site.js

```diff
@@ -1379,14 +1379,14 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions', 'OpenSlid
         $scope.isAllowedToSeeCommentField = function () {
             var isAllowed = false;
             if ($scope.commentsFields.length > 0) {
-                angular.forEach($scope.commentsFields, function(field) {
+                _.find($scope.commentsFields, function(field) {
```

Comment:
This is not what I meant: `_.find` returns the first element where the predicate function returns a truthy value for. If nothing is found, it returns `undefined`. So your predicate function should return something to work properly and to be an improvement compared to the `angular.forEach`:
```javascript
$scope.isAllowedToSeeCommentField = function () {
    var isAllowed;
    if ($scope.commentsFields.length > 0) {
        isAllowed = operator.hasPerms('motions.can_see_and_manage_comments') || _.find(
            $scope.commentsFields,
            function (field) {
                return field.public
            }
        );
    }
    return Boolean(isAllowed);
};